### PR TITLE
WelcomeBack - removing SignalR-Client-Cpp from black listed projects

### PR DESCRIPTION
**Pull request by moozzyk**

### CODE REVIEW

**Comment by msotclas** [comment]
Hi __@moozzyk__, I'm your friendly neighborhood Microsoft Open Technologies, Inc. Pull Request Bot (You can call me MSOTBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <p>The agreement was validated by Microsoft Open Technologies, Inc. and real humans are currently evaluating your PR.</p>

TTYL, MSOTBOT;

**Comment by moozzyk** [comment]
@pranavkm

File: src/SanityCheck/Program.cs [hunk 1]
```diff
@@ -61,7 +61,6 @@ static int Main(string[] args)
                 "SqlClient",
 
                 // Skip newly introduced SignalR packages for beta3
-                "SignalR-Client-Cpp",
                 "SignalR-SqlServer"
```
Comment (pranavkm) [inline src/SanityCheck/Program.cs]: Also remove this + comment?

**Comment by pranavkm** [comment]
:shipit:

**Comment by Eilon** [comment]
:shipit: from me too.

File: src/SanityCheck/Program.cs (hunk 1 above)
Comment (moozzyk) [inline src/SanityCheck/Program.cs]: Let me check is Sitnalr-SqlServer actually passes the check.

Comment (moozzyk) [inline src/SanityCheck/Program.cs]: Signalr-SqlServer works - updated the PR accordingly

**Pull request merged**